validators/errors.js

```diff
@@ -14,11 +23,26 @@ export class WrongCredential extends Error {
   }
 }
 
-export const extractErrors = (error) => {
-  const { errors } = error;
-  const extractedMessages = [];
+export class CustomValidationError extends Error {
+  constructor(message = '', ...args) {
+    super(message, ...args);
+    this.message = extractErrors(message);
+    this.code = 422;
+  }
+}
 
-  errors.map((error) => extractedMessages.push(error.message));
+export class CustomUniqueConstraintError extends Error {
+  constructor(message = '', ...args) {
+    super(message, ...args);
+    this.message = message;
+    this.code = 403;
```

Comment:
Could you think of another error code here. `Error 403: "The server understood the request, but is refusing to authorize it."`